x-pack/plugins/cloud_defend/public/components/control_general_view/index.tsx

```diff
@@ -254,27 +254,22 @@ export const ControlGeneralView = ({ policy, onChange, show }: ViewDeps) => {
   );
 
   const onSelectorChange = useCallback(
-    (updatedSelector: Selector, index: number) => {
-      const old = selectors[index];
+    (updatedSelector: Selector, updatedSelectorIndex: number) => {
+      const oldSelector = selectors[updatedSelectorIndex];
```

Comment:
Thought naming could be more descriptive, let me know if I should revert or extract such changes into a standalone PR.